docs/README.rst

```diff
@@ -200,10 +200,11 @@ Running a specific test, such as ``test_upload.py`` can be done simply by ``ptes
 Contributing
 ------------
 
-Contribution guidelines can be found via `CONTRIBUTING.md <https://github.com/neurodata/brainlit/blob/master/CONTRIBUTING.md>`_
+We welcome all contributors, and encourage them to follow our contribution guidelines gound in `CONTRIBUTING.md <https://github.com/neurodata/brainlit/blob/master/CONTRIBUTING.md>`_. Issues with the "good first issue" tag are meant for contributors that are either new to open source coding, or new to the package. Additionally, users are encouraged to use issues not only to discuss code-related problems, but for more general discussions about the package.
```

Comment:
> [...] follow our contribution guidelines _gound_ in [...]

I assume _gound_ should be _found_, correct?

Reply:
fixed